admin/admin-block-editor.php

```diff
@@ -32,7 +32,7 @@ public function __construct( &$polylang ) {
 		$this->model     = &$polylang->model;
 		$this->pref_lang = &$polylang->pref_lang;
 
-		new PLL_Block_Editor_Filter_Preload_Paths( array( $this, 'preload_paths' ), 10, 2 );
+		add_filter( 'block_editor_rest_api_preload_paths', array( $this, 'preload_paths' ), 10, 2 );
```

Comment:
This won't work as is as the second parameter is no more a `WP_Post` but a `WP_Block_Editor_Context`.

Reply:
Indeed ... 🙈 
Fixed in 540178ab